docs/guides/architecture-scenarios.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Architecture Scenarios
+description: This guide outlines a number of the common architecture scenarios for building applications with Clerk, their characteristics, and limitations.
+---
+
+# Architecture Scenarios
```

Comment:
We use sentence-casing for headings!

```suggestion
# Architecture scenarios
```

Reply:
@issuedat For me this change isn't appearing in the GH web view, but maybe it's just a glitch? Here's what I see:
![image](https://github.com/clerk/clerk-docs/assets/5156915/682597a7-8781-4cc6-a3eb-9ae4793248c6)

I know sometimes the "files changed" view on GH doesn't update if you force push a commit, so if this is changed locally for you then that's fine 👍

Reply:
You're right @S3Prototype! It looks like I manually changed that after having applied the suggestion. Sorry about that — I pushed a change to update it, here's the preview:

<img width="724" alt="Screenshot 2024-06-03 at 15 03 15" src="https://github.com/clerk/clerk-docs/assets/165281975/577e6c3f-237f-46ef-96e9-17e523a253b7">

Reply:
Awesome, thanks ❤️